setup.py

```diff
@@ -55,8 +55,8 @@
 
 import numpy
 import torch
-from torch.utils.cpp_extension import BuildExtension, CppExtension, \
-    CUDAExtension
+from torch.utils.cpp_extension import BuildExtension, CppExtension, CUDAExtension
+from torch.utils.cpp_extension import CUDA_HOME as cuda_home
```

Comment:
`cuda_home` is unused, it was for debugging

Reply:
removed.